examples/nemo/eg6/dummy.f90

```diff
@@ -0,0 +1,35 @@
+program dummy
```

Comment:
Please add the ususal license and a brief comment summarising the purpose of this file.

Reply:
You could take the opportunity to say this is really just about exercising aspects of the read-only verification library.

Reply:
Done